Cluster Programs

```diff
@@ -0,0 +1,57 @@
+Wednesday 13/06/2018
+
+I begin running the SAS programs.
+
+NE Control_1 
+I made a copy of the original and I'm working on the file called NE Control_1 JV
+
+I change the name of the directory in the firts line to:
+G:\MAX-Filer\Research\Labs-kbuzard-S18 
+then I run the entire program one step at a time.
+Started runing at 10.40 am ended at 10.53 am. If I run the entire program it takes less than 3 minutes. 
+I added the Procedure Print the variable citations at the end of the program to see the result.
+Correctly print a table. 
+
+After running the hole program step by step, I start checking it by printing some of the variables, 
+the program print the tables but it take beetwen 10-15 min per table, if it takes longer than that I cancel the printing. 
```

Comment:
There are two options here. One is to do a more selective proc print, in which you only print a small subset of the observations to look at (this is what we'll likely do when we make documentation in knitr). The second is more common when learning about the data: you can open the dataset directly by pointing and clicking and just scrolling around in it. You may want to sort it to learn more about it, but make sure you're not in edit mode or it will actually change the file. And these sorts can take a long time in this mode, so you may want to write a proc sort, creating a separate working data set that you don't intend to use for anything else that you are just using to explore the data for your own purposes of learning about what's going on and making sure everything is sensible.